### ISSUE TITLE
New Add Meeting button is pushing the Add Team button out of reach

### ISSUE THREAD

**Issue by enriquesanchez** [ParabolInc/parabol#7152]
## Issue - Bug

Looks like the new `Add Meeting` button has increased the height of the dashboard's sidebar causing the `Add Team` button (located at the bottom) to be out of view and reach:

https://user-images.githubusercontent.com/3276087/188498394-db533840-459e-4d1b-bd0a-088a4f7d8490.mp4

There's also been a small number of instances where the `Add Meeting` button does not load and this causes the opposite effect, where the top part of the `Meetings`, `Timeline`, and `Tasks` views is out of reach:

<img width="1404" alt="Captura de Pantalla 2022-09-05 a la(s) 10 27 47 a m" src="https://user-images.githubusercontent.com/3276087/188498793-8b72314d-aec0-4da4-82b8-fb07da08d4a1.png">

- **How serious is the bug?** P1
- **OS:** Unspecified
- **Browser:** Replicated on Firefox, Chrome and Safari (Mac)

### Acceptance Criteria
Users can:
   - See the `Add Team` button and 
   - interact with it

**Estimated effort:** 
1 hour to triage. More if root cause is already identified.

**Comment by 87prashant** [comment]
> There's also been a small number of instances where the Add Meeting button does not load

@enriquesanchez I am not able to reproduce it. Any way for the same?

**Comment by jordanh** [comment]
Here's a video of a repro: https://www.loom.com/share/c9772cb137b54bb1baab5cb6b60887a7

**Comment by enriquesanchez** [comment]
It happened again to me. I already had the app open on a browser tab, then opened a different tab and after a while I came back to Parabol and the button was gone. I had to do a hard refresh to get it back.

I didn't see any significant errors in my browser console.

You'll also notice in @jordanh's Loom video, that whatever is happening messes with the whole layout. I could see some scroll bars in different places that normally should not be there.
